deps.qt/checksums/qtnetworkauth-everywhere-src-5.15.2.tar.xz.sha256

```diff
@@ -0,0 +1 @@
+fcc2ec42faa68561efa8f00cd72e662fbc06563ebc6de1dc42d96bb2997acd85  qtnetworkauth-everywhere-src-5.15.2.tar.xz
```

Comment:
I don't believe that we actually use qtnetworkauth, so you should be able to omit this.